src/syntax/binding_in_syntax.ts

```diff
@@ -1,30 +1,44 @@
 import { BindingScopeEnum } from "../constants/literal_types";
 import { interfaces } from "../interfaces/interfaces";
 import { BindingWhenOnSyntax } from "./binding_when_on_syntax";
+import { BindingScopeScopeFactory as BindingScopeScopeFactoryInterface } from "../scope/binding-scope-scope-factory-interface";
+import { BindingScopeScopeFactory } from "../scope/binding-scope-scope-factory";
 
 class BindingInSyntax<T> implements interfaces.BindingInSyntax<T> {
 
     private _binding: interfaces.Binding<T>;
 
+    public bindingScopeScopeFactoryInterface:BindingScopeScopeFactoryInterface<T> = new BindingScopeScopeFactory<T>()
```

Comment:
Assume ok for public properties that can use for testing given not on the interface